### PR TITLE
Block wallet creation+ops on disclaimer, remove background autowallet

### PR DESCRIPTION
**Pull request by mlsteele**
Related to https://github.com/keybase/keybase/pull/3031. The tests in this patch are compatible with the before and after versions of the server. This patch should land first for CI's sake.

### CODE REVIEW

File: go/client/cmd_wallet.go [hunk 1]
```diff
@@ -10,17 +10,18 @@ func newCmdWallet(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Comman
 	subcommands := []cli.Command{
 		newCmdWalletBalances(cl, g),
 		newCmdWalletCancel(cl, g),
-		newCmdWalletExport(cl, g),
+		newCmdWalletCancelRequest(cl, g),
```
Comment (patrickxb) [inline go/client/cmd_wallet.go]: Thank you for sorting!

File: go/client/stellar_common.go [hunk 2]
```diff
@@ -112,3 +112,19 @@ func printPaymentFilterNote(note string) string {
 func cicmp(a, b string) bool {
 	return strings.ToLower(a) == strings.ToLower(b)
 }
+
+func transformStellarCLIError(err *error) {
```
Comment (patrickxb) [inline go/client/stellar_common.go]: nice

File: protocol/avdl/keybase1/constants.avdl [hunk 3]
```diff
@@ -170,6 +170,7 @@ protocol constants {
     SCStellarBadPrev_3106,
     SCStellarWrongPrimary_3107,
     SCStellarUnsupportedCurrency_3108,
+    SCStellarNeedDisclaimer_3109,
```
[the remark this answers is not in the document]
Comment (mlsteele) [reply protocol/avdl/keybase1/constants.avdl]: Eek. This one is sorta blocking cjb and there's pressure to get it in next release. So I'd like to take 3109 and owe you a constant.

[the remark this answers is not in the document]
Comment (patrickxb) [reply protocol/avdl/keybase1/constants.avdl]: You win this time.

**Pull request merged**